.circleci/config.yml

```diff
@@ -15,20 +15,22 @@ executors:
     docker:
       - image: cimg/ruby:3.1.3-node
         environment:
-          - RAILS_ENV=test
-          - TZ: "Europe/London"
+          RAILS_ENV: test
+          TZ: "Europe/London"
+          CI: true
```

Comment:
is this needed? most (circlci, gitlab, travis from a quick google)  CI/CD platforms set this anyway.